kcidb/db/__init__.py

```diff
@@ -74,7 +74,10 @@ def __init__(self, database):
                                   driver
         """
         assert isinstance(database, str)
+
+        self.database = database
         self.driver = misc.instantiate_spec(DRIVER_TYPES, database)
+        self.database = database
```

Comment:
I think doing this once is enough, and this line has no effect. Please remove. Please also remove the extra blank line you're adding above.